src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,63 @@
+package duke.command;
+
+import java.util.ArrayList;
+
+import duke.data.Storage;
+import duke.data.TaskList;
+import duke.task.Task;
+import duke.util.DukeException;
+import duke.util.Ui;
+
+/**
+ * Executes the command to find a task
+ * @author Jicson Toh
+ */
+public class FindCommand extends Command {
+    private final String action;
```

Comment:
I think this variable name should be in upper case as well?

Reply:
same for this!